### DISCUSSION TITLE
Github teams notifications

### DISCUSSION THREAD

**Discussion by abd00s** [community/community#23703] [New to GitHub]
<p>I belong to a team within my organization. We use teams to assign codeowners so we get assigned PR reviews in rotation.</p>
<p>I used to get notifications on PRs where my review was requested. Now all of a sudden, I get email notifications for all activities where a review has been requested from <em>my team</em>. This is producing a lot of noise and I start missing the notification I actually care about.</p>
<p>On the team page, my notification choice is “ignoring”.</p>
<p>This was not an issue to me until the last few days. Am I missing a setting somewhere? Has something changed?</p>

**Comment by nethgato** [comment]
<p>Hey <a class="mention" href="/abd00s">@abd00s</a> :wave:</p>
<p>Thanks so much for submitting this question; it sounds very frustrating! Though I’m curious; which in which Org is showing this behavior? I see you’re a member of two Orgs, so it would be nice to narrow down the scope of research.</p>
<p>I’d also be curious to know what your notification settings are when you visit:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/settings/notifications" target="_blank" rel="noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/62281706/181078823-8573bbcd-8f82-4fbf-af3a-ef85491819f6.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/settings/notifications" target="_blank" rel="noopener">Build software better, together</a></h3>

<p>GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Do the defined settings not hold true to what you believe was chosen? We might be able to review some audit logs to see if settings were changed but it would be great to narrow things down, first.</p>
<p>Thank you! :bow:</p>

**Comment by abd00s** [comment]
<p>Thanks for getting back to me <a class="mention" href="/nethgato">@nethgato</a>,</p>
<p>The org in question is smile-io.</p>
<p>This is what my settings look like right now:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/5/65c255a26a9fc2d83fb553a3462403cdf47b6117.png" data-download-href="/uploads/short-url/ewcAdYI2Og2u36qh2WurhAvLlgH.png?dl=1" title="Notification settings 2020-10-20 11-41-23" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/9205064/181078825-e5c7c2d8-a138-412f-bcc8-c021b119f12e.png" alt="Notification settings 2020-10-20 11-41-23" data-base62-sha1="ewcAdYI2Og2u36qh2WurhAvLlgH" width="306" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/5/65c255a26a9fc2d83fb553a3462403cdf47b6117_2_306x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/5/65c255a26a9fc2d83fb553a3462403cdf47b6117_2_459x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/5/65c255a26a9fc2d83fb553a3462403cdf47b6117_2_612x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/5/65c255a26a9fc2d83fb553a3462403cdf47b6117_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Notification settings 2020-10-20 11-41-23</span><span class="informations">748×1219 95.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>There’s also a “notification” drop down within the team itself, this is what it looks like:<br>
<a href="https://photos.google.com/share/AF1QipMLNRMMTFsGCtG693Ymoeh0tH9NxUhK17M7VPuk3CEpT5IyG3mBN5ZWMHFmd38H3Q?key=N0syVzI4Qk9RT2VMVDlZbEZ5Sjl6YlVoM2RKdFFR" rel="noopener nofollow ugc">link</a><br>
(i’m only allowed to share 1 media as a new user)</p>
<p>Of note, is that I have not modified either of these in a way that would cause this change. In fact, I haven’t made any changes at all. The only change I made was to select <code>ignoring</code> within the team notification settings <em>after</em> the problem began. It did not resolve it.</p>
<p>From the first screenshot, the general notifications settings, it looks correct, so long as I (me personally, not my team) is the one participating or watching. You know what I mean?</p>
<p>I can give you an example from the email notification I get:<br>
<a href="https://photos.google.com/share/AF1QipPpIMTLfEvEl64rDS5dUTLpzaR2jx3UMkuvfh1tsOX2M2ext6uBsmjAcEkZyFE9mA?key=RjY0NjM0dDk3cjQ1ZW11cHlsR3FCQ3RDQVpaS1pB" rel="noopener nofollow ugc">link</a></p>
<p>This is from a PR where a review was requested from me personally. As you can see, I get duplicate emails: one indicating a review was requested from me, and one indicating a request to my team as codeowners.</p>
<p>Prior to this issue, I never received that latter one.</p>
<p>This example is when I am indeed assigned the review. When I am not, I receive only the latter. &lt;-- This is what is producing so much noise for me.</p>
<p>Thanks so much for looking into this for me :slight_smile: This has thrown my entire workflow sideways :grimacing:<br>
Hope those details help you narrow it down.</p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/abd00s">@abd00s</a> thank you so much for that detail! In fact, that does help narrow things down and then some.</p>
<p>Unfortunately, this is a limitation of how notifications works as has been mentioned in this thread:</p>
<aside class="quote" data-post="7" data-topic="856">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/required-codeowner-pull-requests-and-notifications/856/7">Required CODEOWNER pull requests and notifications</a> <a class="badge-wrapper  bullet" href="/c/how-to-use-git-and-github/21"><span class="badge-category-bg" style="background-color: #F1592A;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    Agree with this. My team is a CODEOWNER for a repo, and it would be nice to avoid the email spam that this results in for all team members.
  </blockquote>
</aside>

<p>While that thread was not explicitly about the excessive email notifications, the users there do raise the concern.</p>
<p>Right now, the best course of action to voice your distaste with how the system functions, is to submit your feedback via this form:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://support-assets.github.com/assets/favicon-3dcd2c977a91b2e3f396b300c5ff0d572295ca632ddb7ba1adcacc1ea68dd5c0.ico" class="site-icon" width="16" height="16">
      <a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">support.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="20" height="20">

<h3><a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">Share feedback - GitHub Support</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>That goes directly to our PM team for review and consideration.</p>
<p>Thank you (again) for clearing up the behavior you see, and expressing how you would like to see these notification systems function going forward. :bow:</p>

**Comment by buremba** [comment]
Is there any workaround as of today? I'm part of multiple organisations and some teams inside each org just to get access to some repos that I consult on. Ideally unless somebody mentions me explicitly, I don't want to get any notifications.
